mediator/src/controllers/patient.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios');
+const FHIR_ROOT_URL = 'http://openhim-core:5001/fhir';
+
+function genereateFHITPatientResource(patient) {
```

Comment:
We'll have to move this to a utility, resource transformation or model transformation section of the code. I think we'll have to sanitise the incoming `patient` data and test the data model.